feature-libs/order/components/order-details/order-detail-reorder/order-detail-reorder.component.html

```diff
@@ -0,0 +1,15 @@
+<ng-container *ngIf="order$ | async as order">
+  <ng-container *ngIf="order.code">
+    <div class="cx-nav row">
+      <div class="col-xs-12 col-md-10 col-lg-8">
+        <button
+          #element
+          class="btn btn-primary"
+          (click)="onReorderClick(order)"
+        >
+          {{ 'reorder.button' | cxTranslate }}
+        </button>
+      </div>
+    </div>
+  </ng-container>
+</ng-container>
```

Comment:
About the reorder button the styling seems off at first glance.  If you compare with other buttons in the my account section, the others aren't wide and centered. the wireframe spec also hints a a smaller button.. Please review the styling to make sure it is consistent with the storefront conventions and wireframe requirements. 

Also, at a higher level, there is a discrepancy between the wireframe spec and the PR about where the button is supposed to be placed on the page.  In the spec, the reorder button is expected to be on the right between the order summary and the item list.  

![reorder-button-01](https://user-images.githubusercontent.com/7065346/213496046-3f969d30-c769-4226-993a-1a70d202aa11.png)

Reply:
Leaving as is as decided by Bill